ui/webui/test/README.rst

```diff
@@ -82,6 +82,28 @@ You can also run a test against Firefox instead of Chromium::
 
 See below for details.
 
+Debug logging
+-------------
+Enable debug messages in an interactive browser in the JS console with:
+```js
+window.debugging = "anaconda"
+```
+
+You can do that in a test as well, after the `Installer.open()` call:
+```js
+self.brower.eval_js('window.debugging = "anaconda"')
+```
+
+This also supports other values, e.g. get verbose logging for `dbus` interactions. See
+[Cockpit documentation](https://github.com/cockpit-project/cockpit/blob/main/HACKING.md#debug-logging-in-javascript-console).
```

Comment:
Meh, needs ReST, not Markdown. See https://github.com/martinpitt/anaconda/blob/debug-messages/ui/webui/test/README.rst#debug-logging

(Will fix, but letting the tests run first)